docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -81,10 +79,8 @@ seriesPeriodDetectFFT(series);
 
 **Returned value**
 
-- A real value equal to the period of series data
-- Returns NAN when number of data points are less than four.
-
-Type: [Float64](../../sql-reference/data-types/float.md).
+- A real value equal to the period of series data. [Float64](../../sql-reference/data-types/float.md).
```

Comment:
`A real value equal to the period of the series data. NaN when the number of data points is less than four. [Float64](../../sql-reference/data-types/float.md).`